graph/TypeGraph.java

```diff
@@ -694,33 +697,21 @@ public class Conclusions {
 
             private final Buffered buffered;
             private final Persisted persisted;
-            private boolean outdated;
```

Comment:
we don't need to store `outdated` on the `Conclusions` index, since we will scan the schema graph for modified types at commit time instead.